README.md

```diff
@@ -1,43 +1,42 @@
 # Mail
```

Comment:
FIRST THE CORE, THEN THE MAIL APP AND LAST BUT NOT LEAST **AMERICA** 😈😅 
![donald-trump-trademarked-a-ronald-reagan-slogan-and-would-like-to-stop-other-republicans-from-using-it](https://cloud.githubusercontent.com/assets/19711361/18051790/ffc6c3ec-6df6-11e6-99b3-770b88ff3f22.jpg)